Functions/Assertions/Should.ps1

```diff
@@ -157,3 +198,19 @@ function Format-Because ([string] $Because) {
 
     " because $($bcs -replace 'because\s'),"
 }
+
+function Invoke-LegacyAssertion($assertionEntry, $shouldArgs, $valueToTest, $file, $lineNumber, $lineText) {
```

Comment:
If this is not needed please delete it.

Reply:
You want me to delete the legacy assertion function? I am down to that :)

Reply:
yup

Reply:
Done